pkgpanda/__init__.py

```diff
@@ -678,9 +712,22 @@ def symlink_all(src, dest):
 
         active_buildinfo_full = {}
 
+        dcos_service_configuration = self._get_dcos_configuration_template()
+
         # Building up the set of users
         sysusers = UserManagement(self.__manage_users, self.__add_users)
 
+        def _get_service_names(_dir):
+            service_files = list(map(os.path.basename,
+                                     glob.glob(_dir + "/dcos.target.wants/*.service") +
```

Comment:
can we iterate / walk the folder and match `.service` ends with ourselves? glob has a lot of very bizzare semantics that don't do quite what people think and change based on how you use it and the context it's run in.